app/src/main/scala/org/alephium/explorer/web/AddressServer.scala

```diff
@@ -45,6 +46,11 @@ class AddressServer(transactionService: TransactionService,
 
   val groupNum = groupSetting.groupNum
 
+  // scalastyle:off magic.number
+  private val maxHourlyTimeSpan = Duration.ofDaysUnsafe(7)
+  private val maxDailyTimeSpan  = Duration.ofDaysUnsafe(3650)
```

Comment:
should be `365`, a left over of while I was testing and did want to be blocked by this.